projects/js-packages/publicize-components/src/components/share-buttons/share-buttons.tsx

```diff
@@ -59,20 +59,24 @@ export function ShareButtons( { buttonStyle = 'icon', buttonVariant }: ShareButt
 					'text' !== buttonStyle ? <SocialServiceIcon serviceName={ networkName } /> : null;
 
 				return (
-					<Button
-						key={ networkName }
-						icon={ icon }
-						variant={ buttonVariant }
-						aria-label={ label }
-						href={ href }
-						target="_blank"
-						rel="noopener noreferrer"
-						onClick={ getOnClick( href, { network: networkName } ) }
-						data-network={ networkName }
-						className={ styles[ networkName ] }
-					>
-						{ 'icon' !== buttonStyle ? label : null }
-					</Button>
+					<div className={ styles.container } key={ networkName }>
+						<Button
+							icon={ icon }
+							variant={ buttonVariant }
+							aria-label={ label }
+							href={ href }
+							target="_blank"
+							rel="noopener noreferrer"
+							onClick={ getOnClick( href ) }
+							data-network={ networkName }
+							className={ styles[ networkName ] }
+						>
+							{ 'text' === buttonStyle ? label : null }
+						</Button>
+						{ 'icon-text' === buttonStyle && (
+							<Text className={ styles.label }>Share on { label }</Text>
+						) }
```

Comment:
Why don't we make the whole button clickable here, instead of just the icon?

Reply:
Good point we can make whole clickable.

Reply:
@manzoorwanijk I just did that to match the designs
<img width="437" alt="CleanShot 2023-09-27 at 12 17 30 png" src="https://github.com/Automattic/jetpack/assets/36671565/b941a87d-3934-4834-9863-ae9c3bf94c7e">

Reply:
`Share on { label }` needs to be translated. I have done it in my commit.